01-intro-to-R.Rmd

```diff
@@ -259,24 +245,77 @@ for function names. It's important to be consistent in the styling of your code
 guides are [Hadley Wickham's](http://adv-r.had.co.nz/Style.html) and
 [Google's](https://google.github.io/styleguide/Rguide.xml).
 
-### Getting help
+### Using named arguments
```

Comment:
@kbroman  Let me know what you think of this section.  Since you mentioned this was something people get confused about, I was trying to address named arguments early on.  I was thinking about adding how they are especially useful when you want to modify just one default argument as well, with something like "Named arguments can be especially useful when modifying just one default argument.  You can list just one named argument and not all of the unchanged default arguments that come before it positionally." But I thought it was a bit confusing.  Appreciate any improvements to make the language clearer as well.